### PR TITLE
Add Jetbrains Hacktoberfest swag

### LINKED ISSUE #682: Jetbrains Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Jetbrains

Q. Where have you seen the announcement (URL)?  
A. : https://www.jetbrains.com/lp/hacktoberfest-2020/

Q. What kind of swag (clothing, stickers, ...)?  
A. : software licenses

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by Archis19**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #682

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -263,11 +263,11 @@
...
-    "description": "Fill in the form and get stickers of Jetbrains products for your whole usergroup, event or company. It's free of charge and sent anywere in the world.",
-    "reference": "https://surveys.jetbrains.com/s3/Get-your-complimentary-JetBrains-stickers",
-    "image": "https://i.imgur.com/3ziSn2P.png",
-    "dateAdded": "2019-06-12T13:05:31.000Z",
-    "tags": ["stickers", "expired"]
+    "description": "Fill in the form and you will get a coupon for a one-month access to 'All Products Pack' subscription of Jetbrains. It's free of charge and can be used by anyone.",
+    "reference": "https://www.jetbrains.com/lp/hacktoberfest-2020/#get-the-tools",
+    "image": "https://i.imgur.com/Lee21sb.png",
+    "dateAdded": "2020-09-18T21:33:20.000Z",
+    "tags": ["Software License", "hacktoberfest"]
```
Comment (vikaspotluri123) [inline data.json:270]:
```suggestion
    "tags": ["software", "hacktoberfest"]
```

wdyt @aslafy-z

Comment (aslafy-z) [reply data.json]: LGTM!

**Comment by Archis19** [comment]
can you please review it now! i have commited the changes

**Review by aslafy-z — APPROVED** [review]
LGTM

**Pull request merged**